ingest/ingest_pipeline.py

```diff
@@ -14,14 +14,17 @@
 # Takes expression file and stores it into Firestore
 
 # Ingest cluster file
-python ingest_pipeline.py ingest_cluster --cluster-file ../tests/data/10k_cells_29k_genes.cluster.txt
+python ingest_pipeline.py ingest_cluster --cluster-file ../tests/data/10k_cells_29k_genes.cluster.txt --ingest-cluster
```

Comment:
It's confusing to see "ingest_cluster" twice can you walk me through why this is needed?